docs/components/components.md

```diff
@@ -15,6 +15,27 @@
 }
 ---
 
+<script type="module">
```

Comment:
Does this code need to be available on all component pages or could it be a script pages import if they're using it?

Reply:
We could break it out but then contributors would have to remember